docs/Developers/Review.rst

```diff
@@ -3,19 +3,30 @@ Reviewing
 
 This document is a guide to coala's review process.
 
+Am I Good Enough to Do Code Review?
+-----------------------------------
+
+**Yes!** If you already fixed a newcomer issue.
+
+Please review at least one Pull Request for every review you receive. Start
+with other people's newcomer issues and advance as you do with your own issues.
+
+You can and should help us review source code. Especially try to review source
+code of others and share what you have learnt with them. You can use acks and
+unacks like everyone else and ``cobot`` allows you to set PRs to WIP even.
+
+Try to be thorough: generally, if you don't find an issue with a pull request,
```

Comment:
i dont understand this paragraph at first glance => a) it is badly stated or b) i havent got enough knowledge of what youre saying here